developing.adoc

```diff
@@ -0,0 +1,46 @@
+
+
+= Developing CodeReady Containers
+
+[[developing-overview]]
+== Overview
+
+The following sections describe how to build the project.
+
+[[develop-prerequisites]]
+== Prerequisites
+
+- Git
+- Make
+- A recent Go distribution (>=1.11)
+
+[NOTE]
+====
+You should be able to develop the project on Linux, Windows, or macOS.
+====
+
+[[set-up-dev-env]]
+== Setting up the development environment
+
+[[cloning-repository]]
+=== Cloning the repository
+
+Get the sources from GitHub:
+
+----
+$ git clone https://github.com/code-ready/crc.git
+----
+
+Note: Do not keep the source code in your $GOPATH as link:https://github.com/golang/go/wiki/Modules[Go modules] will cause the `make` to fail.
+Because link:https://github.com/golang/go/wiki/Modules[Go modules]does not expect the source to be in the $GOPATH.
+
+[[dependency-management]]
+== Dependency management
+
+CRC uses link:https://github.com/golang/go/wiki/Modules[Go modules] for dependency management.
```

Comment:
We should use "CodeReady Containers" rather than the "CRC" acronym in our documentation.